apps/GPTdiscord/utils/helpers/image_analysis_helper.py

```diff
@@ -0,0 +1,51 @@
+import os
+import aiohttp
+import io
+import requests
+import base64
+from PIL import Image
+from dotenv import load_dotenv
+
+load_dotenv()
+openai_api_key = os.getenv("OPENAI_API_KEY")
+
+
+async def encode_discord_image(image_url):
+    async with aiohttp.ClientSession() as session:
+        async with session.get(image_url) as response:
+            if response.status == 200:
+                image_data = await response.read()
+                image = Image.open(io.BytesIO(image_data)).convert("RGB")
+                if max(image.size) > 1000:
```

Comment:
be more generous with line-breaks, it adds to readability

Reply:
Done